### DISCUSSION TITLE
bat file for launching beatoraja with the new releases

### DISCUSSION THREAD

**Discussion by jpoiret** [wcko87/beatoraja-english-guide#30] [beatoraja questions / troubleshooting]
I found out that when using the bat file that is given for the new releases, no IR other than mocha is loaded, at least on the newest version (0.8.2).
You should replace the `java` line with `java -Xms1g -Xmx4g -cp "beatoraja.jar;ir\*" bms.player.beatoraja.MainLoader` instead, to manually load the IRs in the ir directory.

**Comment by wcko87** [comment]
oh, that's true. the bat file in the beatoraja source code doesn't seem to include the IR.
let me see if I can find a better bat file to replace the one I currently recommend on the guide, to minimize the amount of editing the user has to do.

**UPDATE:** The correct .bat files can now be found here: https://github.com/wcko87/beatoraja-english-guide/tree/resources/beatoraja-configs

**Reply by wcko87** [reply]
it also seems like the wrong .bat file might have been included in the 0.8.2 release of beatoraja. I'll report that to the dev.
